src/diffusers/models/transformers/auraflow_transformer_2d.py

```diff
@@ -434,6 +435,7 @@ def forward(
         hidden_states: torch.FloatTensor,
         encoder_hidden_states: torch.FloatTensor = None,
         timestep: torch.LongTensor = None,
+        attention_kwargs: Optional[Dict[str, Any]] = None,
```

Comment:
Actually, sorry for my oversight here. We can call it `joint_attention_kwargs` as that is what we call them in Flux as well.